packages/now-cli/src/util/dev/server.ts

```diff
@@ -522,32 +520,17 @@ export default class DevServer {
       return this.cachedNowConfig;
     }
 
-    const pkg = await this.getPackageJson();
-
-    // The default empty `vercel.json` is used to serve all files as static
-    // when no `vercel.json` is present
-    let configPath = 'vercel.json';
-    let config: NowConfig = {
-      version: 2,
-      [fileNameSymbol]: configPath,
-    };
+    const configPath = getNowConfigPath(this.cwd);
 
-    try {
-      configPath = getNowConfigPath(this.cwd);
-      this.output.debug(`Reading ${configPath}`);
```

Comment:
Should we keep some of these debug messages?

Reply:
They're still there, moved into the `readJsonFile()` function.